pkg/plugin/polaris/plugin.go

```diff
@@ -434,11 +434,15 @@ func (p *plugin) ParseConfigAuditReportData(ctx starboard.PluginContext, logsRea
 		return v1alpha1.ConfigAuditReportData{}, fmt.Errorf("unexpected report results count, got: %d, want: %d", len(report.Results), 1)
 	}
 	for _, pr := range report.Results[0].PodResult.Results {
+		severity, err := v1alpha1.StringToSeverity(pr.Severity)
+		if err != nil {
+			return v1alpha1.ConfigAuditReportData{}, err
```

Comment:
should we fail the report generation for unknown severity ? or maybe we should log the error and `continue`

Reply:
If it's not danger or warning we cannot reliably map it to v1alpha1.Severity enum. This would indicate non-backward compatible change in Polaris output or other bug in our parser.